website/src/theme/Navbar/index.js

```diff
@@ -249,6 +249,7 @@ function Navbar() {
         [styles.navbarHidden]: hideOnScroll && !isNavbarVisible,
       })}>
       <div className="navbar__inner">
+        <img referrerpolicy="no-referrer-when-downgrade" src="https://static.scarf.sh/a.png?x-pxid=8f594acf-9b77-44fb-9475-3e82ead1910c" width={0} height={0} alt=""/>
```

Comment:
what does the uuid in this link signify?

Reply:
It's scarf service specific, tracking id for analytics